public/app/features/alerting/unified/home/GettingStarted.tsx

```diff
@@ -20,16 +20,15 @@ export const getOverviewScene = () => {
   });
 };
 
-export default function GettingStarted({ showWelcomeHeader }: { showWelcomeHeader?: boolean }) {
+export default function GettingStarted() {
```

Comment:
`showWelcomeHeader` wasn't used anywhere else in the codebase from what I could tell